models/objects/physic_objects.py

```diff
@@ -0,0 +1,30 @@
+from pygame.surface import Surface
+
+
+class PhysicObject:
+
+    def __init__(self, position: list[int], velocity=None, acceleration=None,
+                 is_static: bool = False):
+        if acceleration is None:
+            acceleration = [0, 0]
```

Comment:
maybe will be beter to use `pygame.math.Vector2`
(doc: https://www.pygame.org/docs/ref/math.html#pygame.math.Vector2)